data/test-cases/plugins/ms-teams-sync/ui-integration/List_linked_channels.md

```diff
@@ -0,0 +1,51 @@
+---
+# (Required) Ensure all values are filled up
+name: "List of linked channels in RHS"
```

Comment:
Do we need to add a test case when no linked channels are present?

Reply:
added a scenario